lib/widgets/message_list.dart

```diff
@@ -483,6 +483,7 @@ class _MessageListState extends State<MessageList> with PerAccountStoreAwareStat
 
   @override
   void onNewStore() { // TODO(#464) try to keep using old model until new one gets messages
+    model?.dispose();
```

Comment:
If this line is removed, it looks like all tests still pass. There should be a test that would notice if we dropped this.

(I think this is exactly what the test discussed in my previous comment would do.)